src/styles/index.scss

```diff
@@ -1,3 +1,204 @@
+
+    & > h1,
+    & > p {
+      margin: 0 16px;
+      text-align: center;
+    }
+  }
+
+  &__image {
+    background-image: url('../images/imac.jpeg');
+    background-size: cover;
+
+    margin: 32px 19px 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    font-family: serif;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $title-color;
+  }
+
+  &__code {
+    font-family: serif;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+
+    color: $txt-color;
+
+    margin-bottom: 16px;
+  }
+
+  /* #endregion */
+
+  /* #region evaluation */
+  &__evaluation {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+
+    margin-top: 16px;
+  }
+
+  &__stars {
+    display: flex;
+    flex-direction: row;
+
+    margin-right: 17px;
+  }
+
+  &__star {
+    margin-right: 4px;
+    background-image: url('../images/star.svg');
+    width: 16px;
+    height: 16px;
+    background-size: cover;
+
+    cursor: pointer;
+
+    &--1 {
+      background-image: url('../images/star-active.svg');
+      width: 16px;
+      height: 16px;
+      background-size: cover;
+    }
+    &--2 {
+      background-image: url('../images/star-active.svg');
+      width: 16px;
+      height: 16px;
+      background-size: cover;
+    }
+    &--3 {
+      background-image: url('../images/star-active.svg');
+      width: 16px;
+      height: 16px;
+      background-size: cover;
+    }
+    &--4 {
+      background-image: url('../images/star-active.svg');
+      width: 16px;
+      height: 16px;
+      background-size: cover;
+    }
+  }
+
+  &__reviews-title {
+    font-family: sans-serif;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+
+    color: $title-color;
+  }
+
+  /* #endregion */
+
+  /* #region price */
+  &__price-container {
+    display: flex;
+    flex-direction: row;
+
+    margin: 24px 16px 16px;
+  }
+
+  &__price-label {
+    font-family: serif;
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 18px;
+
+    color: $txt-color;
+
+    margin-right: 85px;
+  }
+
+  &__price-value {
+    font-family: serif;
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+
+    color: $title-color;
+  }
+
+  /* #endregion */
+
+  /* #region button */
+  &__buy-button {
+    display: inline-block;
+    width: auto;
+    height: auto;
+    border-radius: 5px;
+    background: #00acdc;
+    border: 0 solid #00acdc;
+    padding: 12px 70px;
+
+    margin: 0 16px 16px;
+    transition: all 0.5s ease;
+
+    &-title {
+      color: white;
+      font-family: serif;
+      font-size: 14px;
+      font-weight: 700;
+      line-height: 16px;
+      text-decoration: none;
+      transition: all 0.5s ease;
+
+      text-align: center;
+    }
+
+    &:hover {
+      border: 1px solid #00acdc;
+      background-color: #fff;
+    }
+
+    &:hover,
+    &:hover &-title {
+      color: #00acdc;
+    }
```

Comment:
The hover styles for `&:hover &-title` might not work as intended due to specificity issues. Ensure that the selector correctly targets the intended element. Consider using a more specific selector if necessary.